pydarn/plotting/fan.py

```diff
@@ -355,6 +384,85 @@ def plot_fov(cls, stid: str, dtime: dt.datetime, ax=None,
         citing_warning()
         return beam_corners_aacgm_lats, beam_corners_aacgm_lons, thetas, rs, ax
 
+    @classmethod
+    def plot_radar_position(cls, stid: int, dtime: dt.datetime,
+                            line_color: str = 'black'):
+        """
+        plots only a dot at the position of a given radar station ID (stid)
+
+        Parameters
+        -----------
+            stid: int
+                Radar station ID
+            dtime: datetime datetime object
+                sets the datetime used to find the coordinates of the
+                FOV
+            line_color: str
+                color of the dot
+                default: black
+
+        Returns
+        -------
+            No variables returned
+        """
+        # Get location of radar
+        lat = SuperDARNRadars.radars[stid].hardware_info.geographic.lat
+        lon = SuperDARNRadars.radars[stid].hardware_info.geographic.lon
+        # Convert to geomag coords
+        geomag_radar = aacgmv2.get_aacgm_coord(lat, lon, 250, dtime)
+        mltshift = geomag_radar[1] - (aacgmv2.convert_mlt(geomag_radar[1],
+                                                          dtime) * 15)
+        # Convert to MLT then radians for theta
+        theta_lon = np.radians(geomag_radar[1] - mltshift)
+        r_lat = geomag_radar[0]
+        # Plot a dot at the radar site
+        plt.scatter(theta_lon, r_lat, c=line_color, s=5)
+        return
+
+    @classmethod
+    def plot_radar_label(cls, stid: int, dtime: dt.datetime,
+                         line_color: str = 'black'):
+        """
+        plots only string at the position of a given radar station ID (stid)
+
+        Parameters
+        -----------
+            stid: int
+                Radar station ID
+            dtime: datetime datetime object
+                sets the datetime used to find the coordinates of the
+                FOV
+            line_color: str
+                color of the text
+                default: black
+
+        Returns
+        -------
+            No variables returned
+        """
+        # Label text
+        label_str = ' ' + SuperDARNRadars.radars[stid]\
+                    .hardware_info.abbrev.upper()
+        # Get location of radar
+        lat = SuperDARNRadars.radars[stid].hardware_info.geographic.lat
+        lon = SuperDARNRadars.radars[stid].hardware_info.geographic.lon
+        # Convert to geomag coords
+        geomag_radar = aacgmv2.get_aacgm_coord(lat, lon, 250, dtime)
```

Comment:
Check for the coordinate system?

Reply:
No coords option in read into plot_fov, yet, which is where these are called, so this might be something better addressed when we include other projections.